docs/conf.py

```diff
@@ -24,7 +24,7 @@
 
 templates_path = ['_templates']
 source_suffix = ['.rst', '.md']
-master_doc = 'index'
+main_doc = 'index'
```

Comment:
```suggestion
root_doc = 'index'
```

maybe ?
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc